lms/djangoapps/teams/views.py

```diff
@@ -877,6 +877,14 @@ def post(self, request):
         except User.DoesNotExist:
             return Response(status=status.HTTP_404_NOT_FOUND)
 
```

Comment:
Note that getting the course_module will change the performance characteristics of this method. We have existing load tests-- it would be good to re-run them and see what the impact is (not that we have a whole lot of a choice...).